src/main/java/Ui.java

```diff
@@ -0,0 +1,58 @@
+import java.util.Scanner;
+
+/**
+ * The Ui class is mainly responsible for reading user inputs and printing different output
+ * to the user based on his or her input.
+ */
+class Ui {
+    /**
+     * Shows the Duke logo when the Duke program is started.
+     */
+    String showWelcome() {
+        return "Hello! I am Duke. What can I do for you today?";
+    }
+
+    /**
+     * Shows a divider line.
+     */
+    void showLine() {
+        System.out.println(Duke.LINE);
+    }
+
+    /**
+     * This method reads the input of the user.
+     *
+     * @return A String identical to the user's input.
+     */
+    String readCommand() {
+        Scanner scan = new Scanner(System.in);
+        String command = scan.nextLine();
+        return command;
+    }
```

Comment:
Consider creating a single scanner object in the main method instead of creating a new object every time the method is called.